extension/chrome/src/js/background.js

```diff
@@ -247,11 +247,15 @@ function pingAllExistingTabs() {
 
 
 // Click on extension icon
-chrome.browserAction.onClicked.addListener(function() {
+chrome.browserAction.onClicked.addListener(handleExtensionClick);
+
+function handleExtensionClick() {
 	chrome.storage.local.get(['currentStashBaseUrl'], function(items) {
-		chrome.tabs.create({'url': `${items.currentStashBaseUrl}/`});
+		if (items.currentStashBaseUrl) {
```

Comment:
If there is no `currentStashBaseUrl` it won't open a tab for it, which will leave the popup open